composer.json

```diff
@@ -12,18 +12,19 @@
     ],
     "require": {
         "php": "^7.1",
-        "symfony/framework-bundle": "~3.4|~4.0|~5.0",
-        "symfony/finder": "~3.4|~4.0|~5.0",
-        "symfony/console": "~3.4|~4.0|~5.0",
-        "symfony/intl": "~3.4|~4.0|~5.0",
-        "symfony/translation":  "~3.4|~4.0|~5.0",
-        "symfony/twig-bundle": "~3.4|~4.0|~5.0"
+        "symfony/framework-bundle": "~4.0|~5.0",
```

Comment:
The min version should actually be `4.1` or higher as the double-colon notation for controllers defined as services has been implemented in 4.1.

I suggest actually bumping to 4.4 as minimum, as older 4.x versions are already EOLed.

Reply:
@monteiro ?

Reply:
@monteiro what do you think should be done here?

Reply:
@stof is right. This was an old discussion on dropping 3.4 support. It will make things easier. If someone wants support, can still use the older versions.